.github/workflows/build.yml

```diff
@@ -433,7 +424,15 @@ jobs:
   failure-notification:
     runs-on: ubuntu-latest
     if: failure()
-    needs: [docker-express, docker-next, prepare, linting, formatting, trigger-deployments]
+    needs:
+      [
```

Comment:
Consider using native yaml list:
```
needs:
  - docker-express
  - docker-next
  ...
```

Reply:
reverted change, not relevant to the issue at hand